tools/run_shellcheck.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+TOOLS_DIR="$(cd "$(dirname "${0}")" && pwd -P)"
+ISTIO_ROOT="$(cd "$(dirname "${TOOLS_DIR}")" && pwd -P)"
+
+EXCLUDES="1009,"
+EXCLUDES="${EXCLUDES}1020,"
```

Comment:
can you comment on these numbers or a link to the documentation for these number?

Reply:
Done.